pytradfri/command.py

```diff
@@ -129,7 +125,7 @@ def combine_data(self, command2: Command | None) -> None:
             command2._data, self._data  # pylint: disable=protected-access
         )
 
-    def __add__(self, other: Command | None) -> Command:
+    def __add__(self, other: Command | None) -> Command[T]:
```

Comment:
Side note: I'm thinking maybe we should deprecate this feature and the combine command feature. We don't seem to use it in the library and I don't see why it should be needed by library users.

Reply:
I don't mind removing unused features to reduce maintenance load.